### DISCUSSION TITLE
Data link tutorial

### DISCUSSION THREAD

**Discussion by elinxie** [grafana/grafana#63907] [General]
This is a general feedback for the documentation but for the Data Link in particular. I was wondering if we can have an example of creating and using a datalink with a real grafana link that we can point to? I and finding myself lost in the documentation because I am not sure what to expect from a data link,

**Comment by gelicia** [comment]
Grafana's play environment has several examples of data links that might help https://play.grafana.org/d/yFWXvUp7z/data-links?orgId=1

https://play.grafana.org/d/uptmjHzGk/data-links-cell-value-datalink?orgId=1

Putting real links in the documentation is not straightforward, as data links show up in the visualizations in different ways and the trickiest part is passing along information from the visualization. If you'd like any other examples, the [community forum](https://community.grafana.com/) may be a good resource for different examples. Otherwise, if you have any other ideas of how to improve the documentation, feel free to make an [issue](https://github.com/grafana/grafana/issues).
